modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java

```diff
@@ -347,4 +347,35 @@ public void s3EndpointHasProperRegion() {
                 .isEqualTo(region);
         }
     }
+
+    public static class WithoutServices {
+
+        @ClassRule
+        public static LocalStackContainer localstack = new LocalStackContainer(
+            LocalstackTestImages.LOCALSTACK_0_13_IMAGE
+        );
+
+        @Test
+        public void servicesUseSingleEndpointUri() {
+            assertThat(localstack.getExposedPorts()).as("A single port is exposed").hasSize(1);
+            assertThat(localstack.getEndpointOverride(Service.SQS).toString())
+                .as("Endpoints for all services are the same")
+                .isEqualTo(localstack.getEndpointOverride(Service.S3).toString());
+        }
+
```

Comment:
It's already covered https://github.com/testcontainers/testcontainers-java/blob/main/modules/localstack/src/test/java/org/testcontainers/containers/localstack/LegacyModeTest.java#L48

```suggestion
```